tests/phpunit/TemplateTest.php

```diff
@@ -211,10 +211,16 @@ public function testTemplateRenaming() {
   }
   
   public function testDoiExpansion() {
-    $text = "{{Cite web | http://onlinelibrary.wiley.com/doi/10.1111/j.1475-4983.2012.01203.x/abstract}}";
+    $text = "{{Cite web | http://onlinelibrary.wiley.com/doi/10.1111/j.1475-4983.2012.01203.x/files.pdf}}";
```

Comment:
Is this a real example?  Don't wiley now use `http://onlinelibrary.wiley.com/doi/pdf/10.1111/j.1475-4983.2012.01203.x`?

Reply:
I'm not seeing the value in testing something that doesn't exist.